### PR TITLE
Responsive design

### PR DESCRIPTION
**Pull request by leinerf**

### CODE REVIEW

File: app/client/stylesheets/_home.scss [hunk 1]
```diff
@@ -306,14 +266,15 @@
...
-            font-size: 2.75em;
+            font-size: 2.50em;
         }
 
         .hero-description {
             font-size: 1.2em;
             margin: 2% 10%;
         }
 
+
```
Comment (leinerf) [inline app/client/stylesheets/_home.scss]: take this out

File: app/client/stylesheets/_home.scss [hunk 2]
```diff
@@ -369,26 +330,10 @@
...
-        }
-
-        .landing-page-logo-container {
-            background-image: url('../assets/images/triangles.png');
-            background-size: cover;
-        }
     }
     /* Tablet */
     @media (min-width: 768px) and (max-width: 991px) {
+
```
Comment (leinerf) [inline app/client/stylesheets/_home.scss]: take this out

File: app/client/views/sponsor/sponsor.html [hunk 3]
```diff
@@ -11,62 +11,141 @@
...
 		</div>
 	</section>
 
+
 	<section id="info" class="center">
-		<div id="info-content" class="ui grid">
-			<div class="title one wide phone four wide tablet three wide computer column">
-				Recap
+		<div id="info-content" class="recap-statistics">	
+			<div class="">
```
Comment (leinerf) [inline app/client/views/sponsor/sponsor.html]: take out the class=""

File: app/client/views/sponsor/sponsor.html [hunk 4]
```diff
@@ -11,62 +11,141 @@
...
-					</div>
-					<div class="two wide phone three wide tablet three wide computer column">
-						<strong>73</strong>
-						<span>projects completed</span>
-					</div>
-					<div class="two wide phone three wide tablet three wide computer column">
-						<strong>9</strong>
-						<span>prize categories</span>
-					</div>
+			<div class="">
```
Comment (leinerf) [inline app/client/views/sponsor/sponsor.html]: take out class=""

File: app/client/stylesheets/_home.scss [hunk 5]
```diff
@@ -306,14 +266,15 @@
 
         .hero-title {
             display: block;
-            font-size: 2.75em;
+            font-size: 2.50em;
```
Comment (leinerf) [inline app/client/stylesheets/_home.scss]: 2.5em

File: app/client/views/sponsor/sponsor.html [hunk 6]
```diff
@@ -11,62 +11,141 @@
...
 		</div>
 		<div class="hero-border">
 			<svg version="1.1" id="circle" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 1920 380" xml:space="preserve" preserveAspectRatio="none">
-				<path fill="#fff" d="M0,309c0,0,340,70.8,960,70.8c620,0.2,960-70.8,960-70.8v71H0" transform="translate(0 1)"></path>
+				<path fill="#e7e7e7" d="M0,309c0,0,340,70.8,960,70.8c620,0.2,960-70.8,960-70.8v71H0" transform="translate(0 1)"></path>
 			</svg>
 		</div>
 	</section>
 
+
```
Comment (leinerf) [inline app/client/views/sponsor/sponsor.html:51]: take this out

File: app/client/views/sponsor/sponsor.html [hunk 7]
```diff
@@ -11,62 +11,141 @@
...
-					</div>
-					<div class="two wide phone three wide tablet three wide computer column">
-						<strong>9</strong>
-						<span>prize categories</span>
-					</div>
+			<div class="">
+				<strong>30%</strong>
+				<span>female attendees</span>
+			</div>
+			<div class="">
```
Comment (leinerf) [inline app/client/views/sponsor/sponsor.html]: take this out

File: app/client/views/sponsor/sponsor.html [hunk 8]
```diff
@@ -11,62 +11,141 @@
...
+				<strong>10</strong>
+				<span>schools</span>
+			</div>
+			<div class="">
```
Comment (leinerf) [inline app/client/views/sponsor/sponsor.html]: take this out

File: app/client/views/sponsor/sponsor.html [hunk 9]
```diff
@@ -11,62 +11,141 @@
...
+				<strong>36</strong>
+				<span>hours</span>
+			</div>
+			<div class="">
```
Comment (leinerf) [inline app/client/views/sponsor/sponsor.html]: take this out

File: app/client/views/sponsor/sponsor.html [hunk 10]
```diff
@@ -11,62 +11,141 @@
...
+				<strong>73</strong>
+				<span>projects completed</span>
+			</div>
+			<div class="">
```
Comment (leinerf) [inline app/client/views/sponsor/sponsor.html]: take this out

File: app/client/views/sponsor/sponsor.html [hunk 11]
```diff
@@ -215,95 +294,10 @@
...
-			<div class="sponsorship-content-title">
-					<i class="fa fa-info-circle" aria-hidden="true"></i> More Info?
-			</div>
-			<div class="sponsorship-content-description">
-				If you want more information, check out our <a id="sponsorship-packet" target="_blank" href="https://s3-us-west-1.amazonaws.com/hackuci2019/Hack2019SponsorshipPackage.pdf">sponsorship packet</a>.
-			</div>
-		</div>
-	</section>
-
+
```
Comment (leinerf) [inline app/client/views/sponsor/sponsor.html]: take this out

File: app/client/stylesheets/_home.scss [hunk 12]
```diff
@@ -28,46 +28,36 @@
         font-size: 2.8em;
         font-weight: 700;
     }
+    
     /* Hero */
     #hero {
-        background-image: linear-gradient(to right, #2a2a2a, #191919);
+        background-image: url('../assets/images/landing_pg_bg_viv.png'), linear-gradient(to right, #2a2a2a, #191919);
```
Comment (chriswongtv) [inline app/client/stylesheets/_home.scss]: can we simplify this file name? like `hero.png`? if we're replacing the old one and it isn't used anywhere else, we can move the file to a folder in the same directory named `legacy` or `deprecated`

File: app/client/stylesheets/_sponsor.scss [hunk 13]
```diff
@@ -262,19 +280,17 @@
 
 	/* Sponsors */
 	#sponsors {
-		padding: 6% 0 5% 0;
+		padding: 3% 5% 3% 5%;
```
Comment (chriswongtv) [inline app/client/stylesheets/_sponsor.scss]:
```suggestion
		padding: 3% 5%;
```

File: app/client/stylesheets/_home.scss [hunk 14]
```diff
@@ -295,6 +253,9 @@
     }
     /* Mobile */
     @media (max-width: 767px) {
+        #hero {
+            background-image: linear-gradient(to right, #2a2a2a, #191919);
+        }
```
Comment (chriswongtv) [inline app/client/stylesheets/_home.scss]: leave a newline below it

File: app/client/stylesheets/_sponsor.scss [hunk 15]
```diff
@@ -28,17 +28,22 @@
...
 		font-weight: 700;
 	}
 
 	/* Hero */
 	#hero {
 		background-image: linear-gradient(to right, #2a2a2a, #191919);
-		background-image: url('../assets/images/triangles.png'), linear-gradient(to right, #2a2a2a, #191919);
-		background-size: cover;
+		background-image: url('../assets/images/sponsorship_header_viv.png'), linear-gradient(to right, #2a2a2a, #191919);
+		background-size: contain ;
```
Comment (chriswongtv) [inline app/client/stylesheets/_sponsor.scss]:
```suggestion
		background-size: cover;
```

File: app/client/stylesheets/_sponsor.scss [hunk 16]
```diff
@@ -28,17 +28,22 @@
...
+		background-repeat: no-repeat;
 		background-position: center center;
 		height: calc(100% + 49px);
+		
+		.hero-container {
+			min-width:50%;
```
Comment (chriswongtv) [inline app/client/stylesheets/_sponsor.scss]:
```suggestion
			min-width: 50%;
```

File: app/client/views/sponsor/sponsor.html [hunk 17]
```diff
@@ -11,62 +11,138 @@
...
+			</div>
+			<div>
+				<strong>9</strong>
+				<span>prize categories</span>
+			</div>
+		</div>
+	</section>
+
+	<section id="sponsorship">
+			<div class="title">
```
Comment (chriswongtv) [inline app/client/views/sponsor/sponsor.html]: fix the double indentation for the remainder of the section

File: app/client/views/sponsor/sponsor.html [hunk 18]
```diff
@@ -215,95 +291,10 @@
...
-				Assisting hackers with your APIs, products, and services fosters relationships with participants. Some of them might well be technology leaders of the future. Let them get to know you as the company that makes the best first impression.
-			</div>
-			<div class="sponsorship-content-title">
-				<i class="fa fa-globe" aria-hidden="true"></i> Branding
-			</div>
-			<div class="sponsorship-content-description">
-				Get your name and brand out to the world! Oftentimes, the average person will not know about your company unless they know their way around the field. Make your presence known before they need to find you.
-			</div>
-			<div class="sponsorship-content-title">
-					<i class="fa fa-info-circle" aria-hidden="true"></i> More Info?
```
Comment (chriswongtv) [inline app/client/views/sponsor/sponsor.html:291]: fix double indentation

File: app/client/views/sponsor/sponsor.html [hunk 19]
```diff
@@ -215,95 +291,10 @@
...
-	</section>
-
+	
 	<section id="footer" class="center">
 		<div id="footer-content">
-			<div class="title">
-				HackUCI 2019
-			</div>
-			<button type="button" ng-click="sponsor()">
-					Interested in Sponsoring?
```
Comment (chriswongtv) [inline app/client/views/sponsor/sponsor.html:305]: fix double indentation

File: app/client/views/sponsor/sponsor.html [hunk 20]
```diff
@@ -11,62 +11,138 @@
 					</div>
 				</div>
 				<div class="hero-title-container">
-					<span class="hero-title">Hello.&nbsp;</span>
-					<span class="hero-title">We are HackUCI.</span>
+					<a href="" target="_blank" class="hero-title btn">
```
Comment (chriswongtv) [inline app/client/views/sponsor/sponsor.html]: is this supposed to link to something? also could we add hover effects to the button, such that the button's background will be filled with the same color of the border while the text color remain white?

File: app/client/views/home/home.html [hunk 21]
```diff
@@ -138,10 +134,8 @@
     </section>
     
     <section id="sponsors" class="center">
-        <div id="sponsors-content">
-            <div class="title">
-                Coming Soon
-            </div>
+        <div id="sponsors-content" class="center">
+            create &middot; connect &middot; inspire
```
Comment (chriswongtv) [inline app/client/views/home/home.html]: let's change this text to a sponsor placeholder

File: app/client/views/home/home.html [hunk 22]
```diff
@@ -1,46 +1,42 @@
...
+                    <div class="logo-header logo-cog">
+                        <img src="../assets/images/logo-cog.svg">
+                    </div>
+                    <div class="logo-header">
+                        <img src="../assets/images/logo-anteater.svg">
                     </div>
+                </div>      
+                <span class="hero-title ui grid mobile only">HackUCI 2019</span>
+                <div class="hero-button-container">
+                    <a href="" target="_blank" class="btn">
```
Comment (chriswongtv) [inline app/client/views/home/home.html]: shall we use the style of the CTA on the sponsors page for this button too? we can move those common css elements to the `global.scss` file too so we don't duplicate the classes

File: app/client/stylesheets/_sponsor.scss [hunk 23]
```diff
@@ -541,7 +540,8 @@
     .col-sm-5ths {
         width: 20%;
         float: left;
-    }
+	}
+
```
Comment (chriswongtv) [inline app/client/stylesheets/_sponsor.scss]: remove extra newline

File: app/client/stylesheets/_sponsor.scss [hunk 24]
```diff
@@ -511,13 +508,15 @@
...
 			padding-bottom: 0 !important;
 		}
+
+		.btn {
+			margin-left: 0;
+		}
 	}
 
 	/* Large Screens */
 	@media (min-width: 1650px) {
```
Comment (chriswongtv) [inline app/client/stylesheets/_sponsor.scss]: we should remove this media query if there's no classes in it

File: app/client/stylesheets/_sponsor.scss [hunk 25]
```diff
@@ -131,74 +150,70 @@
...
+			font-size: 1rem;
+			margin-bottom: 5%;
+		}
+		#winners-content {
+			flex:3;
+			padding: 0 3%;
+		}
 	}
-
+
```
Comment (chriswongtv) [inline app/client/stylesheets/_sponsor.scss]: remove extra spaces

File: app/client/stylesheets/_sponsor.scss [hunk 26]
```diff
@@ -131,74 +150,70 @@
...
-		font-weight: 300;
-	}
-
-	#info-content>.recap-description>span {
-		display: block;
-	}
-
-	#info-content>.recap-description>span+span {
-		padding-top: 0.5em;
+
```
Comment (chriswongtv) [inline app/client/stylesheets/_sponsor.scss]: remove extra space

File: app/client/stylesheets/_home.scss [hunk 27]
```diff
@@ -237,43 +225,13 @@
...
-
-    #sponsors-list > a > img {
-        width: 100%;
-        padding: 0 7%;
-    }
     /* Footer */
     #footer {
-        background-image: url('../assets/images/triangles.png'), linear-gradient(to right, #2a2a2a, #191919);
-        background-position: center;
+        background-image: url('../assets/images/low_poly_strip_viv.png');
```
Comment (chriswongtv) [inline app/client/stylesheets/_home.scss]: let's rename this to something like `footer.png`

File: app/client/stylesheets/_sponsor.scss [hunk 28]
```diff
@@ -28,17 +28,22 @@
...
+		font-size: 2.0em;
 		font-weight: 700;
 	}
 
 	/* Hero */
 	#hero {
 		background-image: linear-gradient(to right, #2a2a2a, #191919);
-		background-image: url('../assets/images/triangles.png'), linear-gradient(to right, #2a2a2a, #191919);
-		background-size: cover;
+		background-image: url('../assets/images/sponsorship_header_viv.png'), linear-gradient(to right, #2a2a2a, #191919);
```
Comment (chriswongtv) [inline app/client/stylesheets/_sponsor.scss]: let's rename this to `hero-sponsor.png`

**Review by chriswongtv — DISMISSED** [review]

File: app/client/views/sponsor/sponsor.html (hunk 20 above)
Comment (leinerf) [reply app/client/views/sponsor/sponsor.html]: Can we link coming soon to something else, like the facebook group, and are we releasing it as coming soon or actual apply button?

File: app/client/views/home/home.html (hunk 21 above)
Comment (leinerf) [reply app/client/views/home/home.html]: Actually, I don't think it's a placeholder.  Because we have a section specifically sponsors

**Review by Jeremyyang920 — APPROVED** [review]

**Pull request merged**